src/rt/lifetime.d

```diff
@@ -1207,7 +1207,7 @@ deprecated unittest
     auto x = new S[10000];
     void* p = x.ptr;
     assert(GC.addrOf(p) != null);
-    delete x;
+    _d_delarray_t(cast(void[]*)&x, typeid(typeof(x[0]))); // delete x;
```

Comment:
Since the test is deprecated, shouldn't we just delete it altogether? If not, maybe we can ditch the deprecated on the unittest.

Reply:
It gives the functions coverage until a time when the compiler has removed the library helper entirely.

Reply:
But can we remove the deprecated?

Reply:
IMO, if the functions going away after the compiler will stop referencing it, then it's still deprecated.